laravel-app/app/Services/LoginService.php

```diff
@@ -7,6 +7,12 @@
 
 class LoginService
 {
+    /**
+     * Check if the user is authenticated.
+     *
+     * @param Request $request
+     * @return bool
+     */
     public function isAuthenticated(Request $request): bool
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/